bitmap-team-TeamName/lib/bmp-file-helper.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const fs = require('fs');
+module.exports = exports = {};
+const filepath = `${__dirname}/../img/${process.argv[2]}`;
+const filepathNewfile = `${filepath.slice(0,filepath.length-4)}-new.bmp`;
+const testFilePath = `${filepath.slice(0,filepath.length-4)}-test.txt`;
+
+exports.getBitMap = function(cb) {
+  fs.readFile(filepath, function(err, data) {
+    if (err) throw err;
+    cb(data);
+  });
+};
+
+exports.newBitMap = function(data) {
+  fs.writeFile(filepathNewfile, data, function(err) {
+    if (err) throw err;
+    console.log('your new file can be found here: ', filepathNewfile);
```

Comment:
@npisciotti1 @radenska @sneakyneeks2 @darms - this looks fine with the exception of the commented line below and the comment block for your `makeTestData` - while this is for testing, it should not be on the `master` branch - best approach is to apply this to a dev branch